LBH.AdultSocialCare.Api/V1/Controllers/HomeCareApproveBrokered/HomeCareApproveBrokeredController.cs

```diff
@@ -0,0 +1,36 @@
+using LBH.AdultSocialCare.Api.V1.Boundary.HomeCareApproveBrokeredBoundary.Response;
+using LBH.AdultSocialCare.Api.V1.UseCase.HomeCareApproveBrokeredUseCase.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LBH.AdultSocialCare.Api.V1.Controllers.HomeCareApproveBrokered
+{
+    [Route("api/v1/home-care-approve-brokered")]
+    [Produces("application/json")]
+    [ApiController]
+    [ApiExplorerSettings(GroupName = "v1")]
+    [ApiVersion("1.0")]
+    public class HomeCareApproveBrokeredController : BaseController
```

Comment:
To keep route hierarchy we can have child routes like these in the format `[Route("api/v1/home-care-packages/ {homeCarePackageId}/approve-brokered")]`

Reply:
Thank you make sense, I will.